Common/Orders/Fees/OrderFee.cs

```diff
@@ -32,7 +33,21 @@ public class OrderFee
         /// <param name="orderFee">The order fee</param>
         public OrderFee(CashAmount orderFee)
         {
-            Value = orderFee;
+            Value = new CashAmount(
+                Math.Abs(orderFee.Amount).Normalize(),
```

Comment:
Interestingly, some exchanges offer rebates, so a negative order fee _is_ possible :)

Reply:
Good point! 👍 Thanks for the review :)